libs/ledger/include/ledger/consensus/stake_manager.hpp

```diff
@@ -146,4 +147,44 @@ void TrimToSize(T &container, uint64_t max_allowed)
 }
 
 }  // namespace ledger
+
+namespace serializers {
+
+template <typename D>
+struct MapSerializer<ledger::StakeManager, D>
+{
+public:
+  using Type       = ledger::StakeManager;
+  using DriverType = D;
+
+  static uint8_t const COMITTEE_SIZE       = 1;
+  static uint8_t const UPDATE_QUQUE        = 2;
+  static uint8_t const STAKE_HISTORY       = 2;
+  static uint8_t const CURRENT_SNAPSHOT    = 3;
+  static uint8_t const CURRENT_BLOCK_INDEX = 4;
+
+  template <typename Constructor>
+  static void Serialize(Constructor &map_constructor, Type const &stake_manager)
+  {
+    auto map = map_constructor(5);
+    //    map.Append(COMITTEE_SIZE, stake_manager.committee_size_);
+    map.Append(UPDATE_QUQUE, stake_manager.update_queue_);
+    map.Append(STAKE_HISTORY, stake_manager.stake_history_);
+    map.Append(CURRENT_SNAPSHOT, stake_manager.current_);
+    map.Append(CURRENT_BLOCK_INDEX, stake_manager.current_block_index_);
```

Comment:
Remove comment? And as things are the map constructor should receive 4 as the arg.

Reply:
Done